README.md

```diff
@@ -1 +1,15 @@
 # hw01
+
+## Self Introduction
+My name is Yuwen Xiong. I am a second-year master student at **UChicago Harris School of Public Policy**. Below is the photo of mine:
+
+![Photo of Yuwen](LinkedIn-profile-smaller.jpg)
```

Comment:
I like the photo you added - really clear and lines up to the text. I think you next time, you can play around with the size some more, but otherwise great photo!

Reply:
Thank you, I will!